pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -284,7 +284,7 @@ func (uc *UpgradeCluster) setNodesToUpgrade(kubeClient kubernetes.Client, resour
 			}
 			currentVersion := uc.getNodeVersion(kubeClient, strings.ToLower(*vm.Name), vm.Tags, true)
 
-			if uc.Force {
+			if uc.Force && !strings.Contains(*(vm.Name), "master") {
```

Comment:
This worked and made it so that if a master was already upgraded to the right version we didn't redo just bc we used `--force`. We might want to keep this going forward.